Index: projects/controlFlow.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package projects;\n\npublic class controlFlow {\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/projects/controlFlow.java b/projects/controlFlow.java
--- a/projects/controlFlow.java	(revision a85b8100decc63d8b81e580739d6d16365f436d0)
+++ b/projects/controlFlow.java	(date 1664313794645)
@@ -1,5 +1,8 @@
 package projects;
 
 public class controlFlow {
+    public static void main(String[] args){
+
+    }
 
 }
Index: projects/diceGame.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/projects/diceGame.java b/projects/diceGame.java
new file mode 100644
--- /dev/null	(date 1664313987361)
+++ b/projects/diceGame.java	(date 1664313987361)
@@ -0,0 +1,89 @@
+package projects;
+
+import  java.util.*;
+public class diceGame {
+    public static int score = 0;
+    public static void main(String[] args) {
+
+        ask();
+
+    }
+
+    public static void ask() {
+        Scanner scannerObject = new Scanner(System.in);
+        System.out.println("Would you like to play the Dice Game? (Yes/No)");
+        String userAnswer = scannerObject.nextLine();
+        String yes = "yes";
+        String no = "no";
+        if (userAnswer.toLowerCase().equals(yes)) {
+            System.out.println("Let's play!");
+            System.out.println("------------");
+            play();
+
+        } else if (userAnswer.toLowerCase().equals(no)) {
+            System.out.println("Okay, see you next time!");
+        } else {
+            System.out.println("Error. Try again?");
+            ask();
+
+        }
+    }
+
+    public static void play() {
+        double randomNum = (Math.random() * 6 + 1);
+        int roundedNum = (int) Math.round(randomNum);
+
+
+        if (roundedNum % 2 == 0) {
+            System.out.println("You win!");
+            score++;
+            System.out.println("Score: " + score);
+
+            Scanner scannerObject = new Scanner(System.in);
+            System.out.println("Play again? Yes/No");
+            String userAnswer = scannerObject.nextLine();
+            String yes = "yes";
+            String no = "no";
+
+            if (userAnswer.toLowerCase().equals(yes)) {
+                System.out.println("Let's play!");
+                play();
+
+            } else if (userAnswer.toLowerCase().equals(no)) {
+                System.out.println("Okay, see you next time!");
+            } else {
+                System.out.println("Error. Try again?");
+                ask();
+
+            }
+        } else {
+            System.out.println("You lose! :(");
+            System.out.println("Score: " + score);
+            Scanner scannerObject = new Scanner(System.in);
+            System.out.println("Play again? Yes/No");
+            String userAnswer = scannerObject.nextLine();
+            String yes = "yes";
+            String no = "no";
+
+            if (userAnswer.toLowerCase().equals(yes)) {
+                System.out.println("Let's play!");
+                play();
+
+            } else if (userAnswer.toLowerCase().equals(no)) {
+                System.out.println("Okay, see you next time!");
+            } else {
+                System.out.println("Error. Try again?");
+                ask();
+
+            }
+            }
+
+
+        }
+    }
+
+
+
+
+
+
